File: CS/CustomRibbonItem.cs
using DevExpress.UI.Xaml.Ribbon;
using DevExpress.UI.Xaml.Ribbon.Internal;

namespace ColosSchemesDemo1 {
    public class CustomRibbonItem : RibbonItemBase, IRibbonItem {
        public CustomRibbonItem() {
            DefaultStyleKey = typeof(CustomRibbonItem);
        }
    }
}

File: VB/CustomRibbonItem.vb
Imports DevExpress.UI.Xaml.Ribbon
Imports DevExpress.UI.Xaml.Ribbon.Internal

Public Class CustomRibbonItem
    Inherits RibbonItemBase
    Implements IRibbonItem
    Public Sub New()
        DefaultStyleKey = GetType(CustomRibbonItem)
    End Sub
End Class
